srcpkgs/chromium-widevine/template

```diff
@@ -3,18 +3,18 @@ _chromeVersion="current"
 _channel="stable"
 
 pkgname=chromium-widevine
-version=72.0.3626.109
+version=72.0.3626.119
 revision=1
+create_wrksrc=yes
 short_desc="Browser plugin designed for the viewing of premium video content"
 maintainer="Benjamin Hoffmeyer <hoffmeyer25@gmail.com>"
-homepage="https://www.google.com/chrome"
 license="chrome"
-only_for_archs="x86_64"
 depends="chromium binutils xz"
+homepage="https://www.google.com/chrome"
 repository=nonfree
-create_wrksrc=yes
 distfiles="https://dl.google.com/linux/direct/google-chrome-${_channel}_${_chromeVersion}_amd64.deb"
-checksum=c8bf514f3c70868120924ee2d4872a0e47dcff315bdf799ba2f98dd7aa85fe2b
+checksum=4513f80f2e858b85d321cd10f7a7cddb3a3aa9f6ac4e903dc0d52493337e3a13
+archs="x86_64"
```

Comment:
archs should be where noarch=yes would be

Reply:
Is it just me, or does xlint not find this?

Reply:
only git xlint finds it